TUTORIAL.md

```diff
@@ -0,0 +1,256 @@
+
+![Signature with two 0-cells](./doc/img/tutorial1.png)
+
+The workspace can be then cleared by clicking `Clear` (bottom-left, keyboard shortcut C).
+Clicking on the left of the generator name, we can edit the generator name to respectively $C$ and $D$. $\LaTeX$ is supported with the usual dollar-sign convention.
+
+![Renaming the 0-cells](./doc/img/tutorial2.png)
+
+## The 1-cells
+
+Now, to create the generating 1-cells, we need to tell the proof assistant their sources and targets. Let us start with $F$, which runs from $C$ to $D$.
+
+First, we select $C$ by clicking on it or pressing the number 1. We can designate it as the source of some future $F$ by clicking `Source` (bottom-left, or clicking S).
+
+Now the blue dot representing $C$ should appears in the bottom-left corner. The proof assistant has stashed the diagram we have chosen for source, and is waiting for us to select, or construct, the target.
+In our case, this is easy, as the target of $F$ is just $D$. So let's click on $D$ and press `Target` (or T).
+This will give a 1-cell, our $F$. We repeat the process with $C$ and $D$ swapped to construct $G$, which runs from $D$ to $C$.
+
+Clicking on our newly constructed $F$, the resulting diagram will be displayed in the workspace. Note that since $F$ is a 1-cell, the diagram displayed will be 1-dimensional, which we read bottom-to-top.
+
+![Viewing our 1-cells](./doc/img/tutorial3.png)
+
+In this workspace view, we also have some slice-controls on the right, which we'll explain how to use later on. First, let us rename these 1-cells to $F$ and $G$, as we have done previously for $C$ and $D$.
+
+## The 2-cells
+
+Adding the 2-cells $\alpha$ and $\beta$ will be trickier for two reasons:
+- Their sources and targets are not generating 1-cells, but composites of 1-cells.
+- They have to be invertible.
+
+Recall that $\alpha$ has source the identity on $1_C$ and target the composite of $F$ and $G$. Visually, it looks like a cup.
+To construct the source, select the generator $C$ and click on `Identity` (or I): the blue point gets promoted to a blue line that we may take as the `Source` of our 2-dimensional $\alpha$.
+
+![Taking the identity on C as source](./doc/img/tutorial4.png)
+
+For the target, we select $F$ and then paste $G$ on top of it by clicking towards the upper edge. Since there is only one generator that $D$ as a source, namely $G$, it is automatically selected by the tool.
+
+![Constructing the composite of F and G](./doc/img/tutorial5.png)
+
+Now, this composite diagram we just constructed has the same boundary as the identity on $C$, which we selected as a source (they both have source and target $C$). This is a necessary condition for the tool to allow us construct a new generator in the signature called *globularity*.
+
+For example, if we didn't compose $G$ on top of $F$, the globularity check would have failed (as the target of $F$ is $D$ and not $C$) and `Target` button would have been hidden by the tool.
```

Comment:
The `Target` button won't actually be hidden. I changed that a while ago so that Source/Target are visible even if the globularity check fails (1 so we avoid running the check unnecessarily, and 2 because it's more intuitive if you get an error message explaining why the globularity check failed).